### DISCUSSION TITLE
忘记蓝灯专业账号的邮箱了，能否用当时注册的手机号登录。

### DISCUSSION THREAD

**Discussion by Zhangsw2024** [getlantern/lantern-forum-cn#871] [Q&A]
忘记蓝灯专业账号的邮箱了，能否用当时注册的手机号登录。

**Comment by MaxMKMax** [comment]
@Zhangsw2024 您好. 只是能用邮箱登陆. 还是您有支付单据吗? 我让技术部门帮您查您的账号邮箱.

**Comment by Zhangsw2024** [comment]
***@***.******@***.***，谢谢。发自我的 iPad在 2024年9月17日，23:08，Mike K ***@***.***> 写道：﻿
@Zhangsw2024 您好. 只是能用邮箱登陆. 还是您有支付单据吗? 我让技术部门帮您查您的账号邮箱.
